src/main/java/buildcraftAdditions/utils/BlacklistedTank.java

```diff
@@ -0,0 +1,50 @@
+package buildcraftAdditions.utils;
+
+import java.util.ArrayList;
+
+import net.minecraft.tileentity.TileEntity;
+
+import net.minecraftforge.fluids.Fluid;
+import net.minecraftforge.fluids.FluidStack;
+
+/**
+ * Copyright (c) 2014-2015, AEnterprise
+ * http://buildcraftadditions.wordpress.com/
+ * Buildcraft Additions is distributed under the terms of GNU GPL v3.0
+ * Please check the contents of the license located in
+ * http://buildcraftadditions.wordpress.com/wiki/licensing-stuff/
+ */
+public class BlacklistedTank extends RestrictedTank {
```

Comment:
indeed that's what i want, you are using this tank in the lavaCoil so only lava should go in there